integreat_cms/static/src/js/media-management/component/edit-sidebar.tsx

```diff
@@ -221,6 +229,21 @@ const EditSidebar = ({
                         <label className="secondary my-0">{mediaTranslations.label_file_modified}</label>
                         <p>{file.lastModified}</p>
                     </div>
+                    {file.isGlobal && (
+                        <div className="flex flex-wrap justify-between gap-2 hover:bg-gray-50 p-4 border-b">
+                            <label className="secondary my-0">{mediaTranslations.label_file_is_hidden}</label>
```

Comment:
```suggestion
                            <label className="secondary my-0" for="file-hide-input">{mediaTranslations.label_file_is_hidden}</label>
```
I think clicking the text should toggle the checkbox